Libs/Gtk-3.0/Classes/CssProvider.cs

```diff
@@ -2,37 +2,32 @@
 using System.Runtime.InteropServices;
 using System.Text;
 using GLib;
+using GLib.Native;
 using GObject;
+using GObject.Native;
 
 namespace Gtk
 {
-    /*
     public partial class CssProvider
     {
-        public CssProvider() { }
+        public static CssProvider New()
+            => ObjectWrapper.WrapHandle<CssProvider>(Native.CssProvider.Instance.Methods.New(), false);
```

Comment:
The wrapper checks if an object is already known for this pointer and afterwards calls the constructor per reflection. But as we are creating a new instance we can omit this check as we now there is no other instance and direcly call a constructor.

Reply:
How would this work? Would we just add a `WrapNewHandle<>()` that bypasses all the checks?

Reply:
It would be as easy as this:

```cs
public static CssProvider New()
    => new CssProvider(Native.CssProvider.Instance.Methods.New(), false);
```